src/Microsoft.Azure.WebJobs.Host/Blobs/ParameterizedBlobUrl.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using Microsoft.Azure.WebJobs.Host.Bindings.Path;
+
+namespace Microsoft.Azure.WebJobs.Host.Blobs
+{
+    internal class ParameterizedBlobUrl : IBindableBlobPath
```

Comment:
>ParameterizedBlobUrl [](start = 19, length = 20)

Can you add a comment like:
This handles values like "{data.url}" which are expected to resolve to a full blob URL.